jkube-kit/build/api/src/main/java/org/eclipse/jkube/kit/build/api/helper/DockerFileUtil.java

```diff
@@ -209,4 +273,36 @@ private static File getHomeDir() {
         return new File(homeDir);
     }
 
+    private static void updateMapWithArgValue(Map<String, String> result, Map<String, String> args, String argString) {
+        if (argString.contains("=") || argString.contains(":")) {
+            String[] argStringParts = argString.split("[=:]");
+            String argStringValue = argString.substring(argStringParts[0].length() + 1);
+            if (argStringValue.startsWith("\"") || argStringValue.startsWith("'")) {
+                // Replaces surrounding quotes
+                argStringValue = argStringValue.replaceAll("^\"|\"|'|'$", "");
```

Comment:
This Regex is dangerous, and reported as a bug https://sonarcloud.io/project/issues?id=jkubeio_jkube&open=AXVraCJxz5kaBwk-uluS&pullRequest=175&resolved=false&types=BUG